cmake/target_toolchain.cmake

```diff
@@ -42,20 +42,20 @@ endif()
 unset(CMAKE_C_COMPILER)
 unset(CMAKE_C_COMPILER CACHE)
 
-# Configure the toolchain based on what toolchain technology is used
-# (gcc, host-gcc etc.)
+# A toolchain consist of a compiler and a linker.
```

Comment:
I think that's a bit too much of a simplification. Looking for instance at the Zephyr SDK: - as, - cpp, - strip, - nm, - objdump, - ar, - ranlib, - gdb, - libgcc.a,  - crtfastmath.o, - crt*.o,
Suggest: "the two main components of a toolchain are the compiler and the linker."

Maybe sneak that rephrase in a Part N not merged yet?

PS: I'm usually the one complaining about how bundling unrelated changes in the same commit breaks cherry-pick etc., however separate commits for small whitespace/comment fixes is maybe it a bit too far?

Reply:
Good comment and you are right. These other tools of the toolchain you mention have also been thought of; the will get their own bintools or binutils category of their own.